device/CMakeLists.txt

```diff
@@ -30,3 +30,17 @@ set_target_properties(umd_device PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib
     POSITION_INDEPENDENT_CODE ON
 )
+
```

Comment:
This should also be used by build-tests.yml and then consumed by run-tests.yml?
I had some problems while trying to make it work the first time...

I tried copying the same config as https://github.com/tenstorrent/tt-mlir/blob/main/.github/workflows/build-and-test.yml